customer-terminate-service/app/src/main/java/se/inera/intyg/cts/application/task/ExportTask.java

```diff
@@ -0,0 +1,24 @@
+package se.inera.intyg.cts.application.task;
+
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+import org.springframework.scheduling.annotation.Scheduled;
+import se.inera.intyg.cts.application.service.ExportService;
+
+public class ExportTask {
+
+  private static final String TASK_NAME = "CollectCertificateTextsTask.run";
+  private static final String LOCK_AT_MOST = "4m";
```

Comment:
The exporting of package includes creating a encrypted zip file and (later) uploading the file with metadata to sjut. This can potentially take longer time than the other tasks and therefor it has a different lock-length.